python/ray/dag/compiled_dag_node.py

```diff
@@ -2281,11 +2298,13 @@ def visualize(
 
         # Dot file for debuging
         dot = graphviz.Digraph(name="compiled_graph", format=format)
-
+        # Give every actor a unique color (Colors between 24k -> 40k seem readable)
```

Comment:
Did you imply showing up to 16 colors should be OK, but the coloring scheme does not scale well beyond that? If so, please add that to the comment explicitly.

Reply:
So the coloring scheme will keep going infinitely and wrap around, but the text is always black, and the colors could end up being too dark so the text isn't as visible, will add that info in comment